### PR TITLE
Add card for access override status

### LINKED ISSUE #878: Show a card on the dashboard if the user has an override
Show both current and past overrides and instructions on how to renew.

### PR DESCRIPTION
**Pull request by JoshOldenburg**
image
Fixes #878.

### CODE REVIEW

**Comment by JoshOldenburg** [comment]
image
This is an alternative, if the welcome card is made the same width as the others.

File: resources/views/welcome.blade.php [hunk 1]
```diff
@@ -64,6 +64,21 @@
...
+            <div class="col-sm-6 col-md-3 col-lg-4">
+                <div class="card">
+                    <div class="card-body">
+                        <h4 class="card-title">
+                            Access Status
+                        </h4>
+                        <p class="card-text">
+                            While you do not have an active dues payment, you have a temporary access override to
+                            various RoboJackets-managed IT systems that will expire on {{ $overrideDate }}.
+                        </p>
```
Comment (kberzinch) [inline resources/views/welcome.blade.php]: Suggested phrasing change:
>**Access Override**
>While you have not paid dues for this semester, you have temporary access to RoboJackets systems until {{ $overrideDate }}. If you have questions or need an extension, please ask in #it-helpdesk.

File: app/Http/Controllers/DashboardController.php [hunk 2]
```diff
@@ -50,13 +50,18 @@ public function index(Request $request): View
             $firstPayment = null;
         }
 
+        $hasOverride = $user->access_override_until && $user->access_override_until > now();
```
Comment (kberzinch) [inline app/Http/Controllers/DashboardController.php]: Could we also show a card if an override has expired recently (say, 2 weeks?)

**Comment by kberzinch** [comment]
Let's also make the Welcome card narrower to match.

**Comment by JoshOldenburg** [comment]
image
image

**Pull request merged**